configuration.nix

```
@@ -48,6 +48,7 @@ isMaximal: {
 
       # Assembly is not common, and the asm LSP is a major hit-or-miss
       assembly.enable = false;
+      astro.enable = isMaximal;
```

```suggestion
      astro.enable = false;
```

It's not needed in maximal.